owncloud/phoenix
# Bugfix/show images media viewer

## Description
Due to changes while introducing public links media viewer broke :man_shrugging: 

## How Has This Been Tested?
- upload some images
- open them in media viewer
- see images

## Types of changes

## Checklist:

## Open tasks:
